README.md

```diff
@@ -54,34 +54,6 @@ The Google Client uses a few methods to determine how it should authenticate wit
 4. If running in **Google App Engine**, the built-in service account associated with the application will be used.
 5. If running in **Google Compute Engine**, the built-in service account associated with the virtual machine instance will be used.
 
-### Public URLs
```

Comment:
I would just put this back and clarify that older versions will not work. Having everything in a single branch is easier to maintain imo.